packages/radix-vue/src/Calendar/Calendar.test.ts

```diff
@@ -252,14 +252,14 @@ describe('calendar', async () => {
 
     const nextButton = getByTestId('next-button')
 
-    expect(getByTestId('date-1-1')).toHaveTextContent('1')
+    expect(getByTestId('date-8-1')).toHaveTextContent('1')
```

Comment:
@epr3 Please notice this test case starts now in August, so this `expect` is checking only that August 1st is visible. The issue was that September 1st was skipped. Another `expect` is checking September, but that is September 2025 (because of `for` loop), which does not start on Sunday so the error would not happen there. I think you could move second `expect` to replace the first one. Probably adding a comment will make it easier for someone in the future to understand what was the intent here.

If you would like to further improve this test speed you could consider leaving only one `for` loop. I've calculated that there will be max 13 months at any point between situations that trigger this issue, so maybe `i < 14` will cover enough months for what was originally tested during those 36 months verified currently.